tests/test_image/test_io.py

```diff
@@ -190,6 +190,22 @@ def test_imread(self):
         img_pil_exif_unchanged = mmcv.imread(
             self.exif_img_path, backend='pillow', flag='unchanged')
         assert img_cv2_exif_unchanged.shape == img_pil_exif_unchanged.shape
+        img_cv2_color_ignore_exif = mmcv.imread(
+            self.exif_img_path, flag='color_ignore_orientation')
+        img_pil_color_ignore_exif = mmcv.imread(
+            self.exif_img_path,
+            backend='pillow',
+            flag='color_ignore_orientation')
+        assert img_cv2_color_ignore_exif.shape == (300, 400, 3)
```

Comment:
What's the shape of img_cv2_exif  at L185? You may add an assertion on that as well